### PR TITLE
Support Omron in BP Viz

### PR DESCRIPTION
**Pull request by reneefromhold**
## Overview

Support Omron HR Data in BP Visualization

## Security

REMINDER: All file contents are public.

- [ x] I have ensured no secure credentials or sensitive information remain in code, metadata, comments, etc.
  - [x ] Please verify that you double checked that .storybook/preview.js does not contain your participant access key details. 
  - [x ] There are no temporary testing changes committed such as API base URLs, access tokens, print/log statements, etc.
- [ x] These changes do not introduce any security risks, or any such risks have been properly mitigated.

Describe briefly what security risks you considered, why they don't apply, or how they've been mitigated.

### Testing
- [ ] MyDataHelps iOS app
- [ ] MyDataHelps Android app
- [ x] [MyDataHelps website](mydatahelps.org)

### Documentation
- [ x] I have added relevant Storybook updates to this PR as well.
- [ ] If this feature requires a developer doc update, tag @CareEvolution/api-docs.

Consider "Squash and merge" as needed to keep the commit history reasonable on `main`.

## Reviewers

Assign to the appropriate reviewer(s). Minimally, a second set of eyes is needed ensure no non-public information is published. Consider also including:
- Subject-matter experts
- Style/editing reviewers
- Others requested by the content owner

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@babel/preset-env": "^7.21.5",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.5",
-    "@careevolution/mydatahelps-js": "^3.10.0",
```
Comment (brandoncharnesky) [inline package.json:33]: Is this bump intentional?

File: src/helpers/blood-pressure-data-providers/combined-blood-pressure-data-provider.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { BloodPressureDataPoint, SurveyBloodPressureDataParameters } from "."
 import surveyBloodPressureDataProvider from "./survey-blood-pressure-data-provider"
 import deviceDataBloodPressureDataProvider from "./device-blood-pressure-data-provider"
 
-export type BloodPressureDeviceDataSource = 'AppleHealth' | 'GoogleFit';
```
Comment (brandoncharnesky) [inline src/helpers/blood-pressure-data-providers/combined-blood-pressure-data-provider.ts:5]: nit:  quote marks not matching

**Review by brandoncharnesky — COMMENTED** [review]
How are you generating test data for this?

**Comment by reneefromhold** [comment]
> How are you generating test data for this? I am working on updates for a different data provider and unsure the best practices.

I have purchased an omron device and am using data directly pulled in

**Review by brandoncharnesky — APPROVED** [review]
Looks good!  🚢

**Pull request merged**